src/config/consts.js

```diff
@@ -2,6 +2,43 @@ const ONCE_A_DAY = 'ONCE_A_DAY';
 const ONCE_PER_HOUR = 'ONCE_PER_HOUR';
 
 export const consts = {
+  a11yLabel: {
+    address: 'Adresse',
+    backIcon: 'Zurück Taste',
+    backIconHint: 'Navigieren zurück zur vorherigen Seite',
+    bookmarkList: 'Lesezeichenliste (Taste)',
+    bookmarkListHint: 'Zu der Lesezeichenliste hinzufügen',
+    button: 'Taste',
+    category: 'Kategorie',
+    closeMenuIcon: 'Schließen Taste',
+    closeMenuHint: 'Menü schließen',
+    heading: 'Überschrift',
+    image: 'Bild',
+    imageCarousel: 'Bild des Bildkarussells',
+    imageCarouselLinkHint: 'Navigiert zum Gegenstand',
```

Comment:
we do not know exactly where it goes here, so lets have no hint in this case where it is used now

Reply:
done in 71c498a949d6f531126e672bc262773d65a2cbf2